xen_vm_network.py

```diff
@@ -5,9 +5,11 @@
 # Copyright (c) 2015, InnoGames GmbH
 
 from __future__ import print_function
-import socket, time
+import socket
+import time
 import subprocess
 
+
 def resolve_to_vserver(vifname=False, xmdata=False):
```

Comment:
I am wondering why using defaults at all here? The parameters have to be passed in.